src/outofprocerrorwrapper.ts

```diff
@@ -0,0 +1,8 @@
+import { OrchestratorState } from "./classes";
+
+export class OutOfProcErrorWrapper extends Error {
+    constructor(state: OrchestratorState) {
+        const message = JSON.stringify(state);
```

Comment:
I should probably improve the Error, as this message and stacktrace need to serve two purposes:

1. Embed the OrchestratorState data for the C# extension to read. This data should be easy to grab with regex or other string parsing tools in a unique way, so that custom Errors thrown by customer code will not interfere with extracting this data.
2. Be human readable with clarity of how to resolve the issue, as this is the exception that will be logged in AppInsights.

Proposed error string:
```
Message: <Original-Error-Message>

$OrchestratorOutOfProcState$<Orchestrator-State-Json>$OrchestratorOutOfProcState$
StackTrace: <Original-Error-StackTrace>
```